palantir/datasets/rpc/catalog.py

```diff
@@ -43,7 +44,7 @@ def create_dataset(
         _json: Any = ConjureEncoder().default(request)
 
         _path = "/catalog/datasets"
-        _path = _path.format(**_path_params)
+        _path = format_path_with_params(_path, _path_params)
```

Comment:
given this is codegen surely this must also be an issue in [conjure-python](https://github.com/palantir/conjure-python). Can you raise an issue (or better, port this fix) over there?

Reply:
Oh, didn't know this was codegen. I'd also advocate for a header to be added on top of codegen files indicating as such.